airdrops/src/airdrops.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "id": "1",
+    "title": "Airdrop #1",
+    "description": "A mysterious collection of digital artifacts found in an abandoned crypto wallet. Legend says they multiply when no one is watching.",
+    "contractAddress": "0x1234567890123456789012345678901234567890",
+    "tokenAmount": "5000",
+    "tokenSymbol": "UP",
+    "isEligible": true
```

Comment:
Please change this to a `recipientsFile` with a URL and add somewhere a `isElligible(address)` function that would return a number of tiokens for which an address is elligible (to be implemented later!)